docs/_data/nav_docs.yml

```diff
@@ -41,3 +41,5 @@
       title: ReactDOM
     - id: react-dom-server
       title: ReactDOMServer
+    - id: pure-render-mixin
```

Comment:
This page should be part of the addons docs. They're going to be grouped rather than top level, do I understand it right?